nifi/1.10/0.1.0-1.5.0/index.md

```diff
@@ -0,0 +1,46 @@
+This nifi README may be used as a starting point for writing a Service Guide for your DC/OS Service.
```

Comment:
remove this line

Reply:
Removed the line as suggested, in master branch of https://github.com/debangshuk/examples